core/server/helpers/authors.js

```diff
@@ -6,27 +6,31 @@
 // By default, authors are separated by commas.
 //
 // Note that the standard {{#each authors}} implementation is unaffected by this helper.
-const proxy = require('./proxy'),
-    _ = require('lodash'),
-    urlService = require('../services/url'),
-    SafeString = proxy.SafeString,
-    templates = proxy.templates,
-    models = proxy.models;
-
-module.exports = function authors(options) {
-    options = options || {};
-    options.hash = options.hash || {};
+const proxy = require('./proxy');
+const _ = require('lodash');
+const urlService = require('../services/url');
+const {SafeString, templates, models} = proxy;
 
-    const autolink = !(_.isString(options.hash.autolink) && options.hash.autolink === 'false'),
-        separator = _.isString(options.hash.separator) ? options.hash.separator : ', ',
-        prefix = _.isString(options.hash.prefix) ? options.hash.prefix : '',
-        suffix = _.isString(options.hash.suffix) ? options.hash.suffix : '',
-        limit = options.hash.limit ? parseInt(options.hash.limit, 10) : undefined,
-        visibilityArr = models.Base.Model.parseVisibilityString(options.hash.visibility);
+module.exports = function authors(options = {}) {
+    options.hash = options.hash || {};
```

Comment:
Didn't use nested destructuring here because the input can be considered untrusted